packages/core/src/agent/index.ts

```diff
@@ -279,6 +279,22 @@ export class Agent<
     });
   }
 
+  private async getResolvedOutputProcessors(runtimeContext?: RuntimeContext): Promise<OutputProcessor[]> {
+    if (!this.#outputProcessors) {
+      return [];
+    }
+
+    if (typeof this.#outputProcessors === 'function') {
+      if (!runtimeContext) {
+        // Create a default runtime context if none provided
+        return [];
+      }
```

Comment:
The comment contradicts what the code is doing, don't we want to return the output processors even if there's no runtime context